src/System.ServiceModel.Duplex/src/System.ServiceModel.Duplex.csproj

```diff
@@ -5,7 +5,7 @@
     <AssemblyName>System.ServiceModel.Duplex</AssemblyName>
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
```

Comment:
I know it's not the change of this PR, but S.SM.Duplex is _not_ a partial façade assembly.